doc/riverctl.1.scd

```diff
@@ -0,0 +1,130 @@
+RIVERCTL(1) "github.com/ifreund/river" "General Commands Manual"
+# NAME
+
+riverctl - command-line interface for controlling river
+
+# SYNOPSIS
+
+*riverctl* *close*|*declare-mode*|*enter-mode*|*exit*|*focus-output*|
+*focus-view*|*layout*|*map*|*mod-master-count*|*mod-master-factor*|
+*send-to-output*|*set-focused-tags*|*set-view-tags*|*spawn*|*toggle-float*|
+*toggle-focused-tags*|*toggle-view-tags*|*zoom* [_command specific arguments_]
+
+# DESCRIPTION
+
+*riverctl* is a command-line interface inspired by bspc from bspwm
+used to control and configure river.
```

Comment:
Readers of the man page do not really care what the inspiration was. The man page should describe what a program is without falling back to "it's just like $other_program".

Reply:
Fair point.